docs/message-history.md

```
@@ -257,13 +259,27 @@ result1 = agent.run_sync('Tell me a joke.')
 result2 = agent.run_sync('Explain?', message_history=result1.all_messages())
 
 assert result1.conversation_id == result2.conversation_id
+assert result1.run_id != result2.run_id
 ```
 
-To override or fork:
+```python {title="pass a pre-minted run_id"}
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)